packages/ui/src/theme/css/component/phoneNumberField.scss

```diff
@@ -1,3 +1,33 @@
 .amplify-phonenumberfield select:not(:focus) {
   border-right: none;
 }
+
+.amplify-phonenumberfield {
+  //here we need to override both the selectfield and textfield variables because the phonenumberfield is composed of both of these
+  //the initial value is being pulled from the fieldcontrol values but we cannot set fieldcontrol values here because selectfield and textfield both overwrite those
+  --amplify-components-selectfield-color: var(
```

Comment:
This is the first time we've run into a case where both the parent and child components are relying on the same underlying design tokens (in this case fieldcontrol), in this case we need to chain the design tokens together having the parent overwrite the child and the child overwrite the root, otherwise the parents changes will be overridden by the child component.